### PR TITLE
[KO] Update translation

### PR DESCRIPTION
**Pull request by marocchino**
@Dalgona, @malkoG, @devleoper, @riseshia review please.

FYI. I use this command for diff:

~~~
git diff f8baebe..bcb0548 -- lessons/
~~~

Also, includes this [review](https://github.com/doomspork/elixir-school/pull/726#discussion_r83355310).

### CODE REVIEW

**Comment by riseshia** [comment]
gorapaduck

File: ko/lessons/basics/sigils.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Elixir에서는 리터럴을 표현하거나 리터럴을 가지고 작업을 
   - `~s`는 이스케이프나 내부 식 전개를 **하면서** 문자열을 생성합니다
   - `~W`는 이스케이프나 내부 식 전개 **없이** 단어 리스트를 생성합니다
   - `~w`는 이스케이프나 내부 식 전개를 **하면서** 단어 리스트를 생성합니다
+  - `~N`은 `NaiveDateTime` 구조체를 생성합니다.
```
Comment (devleoper) [inline ko/lessons/basics/sigils.md]: #750

File: ko/lessons/advanced/otp-supervisors.md [hunk 2]
```diff
@@ -42,6 +42,18 @@ children = [
...
+자식 프로세스의 충돌을 처리하는 방법은 여럿 있습니다.
+
++ `:permanent` - 자식은 언제나 재시작합니다.
+
++ `:temporary` - 자식을 재시작하지 않습니다.
+
++ `:transient` - 자식이 비정상적으로 종료되었을 때만 재시작합니다.
+
+필수 옵션은 아니며, 기본 값은 `:permanent`입니다.
+
```
Comment (devleoper) [inline ko/lessons/advanced/otp-supervisors.md]: #756

**Comment by malkoG** [comment]
💯

**Pull request merged**